lib/defaultMetrics.js

```diff
@@ -28,6 +28,9 @@ var metrics = {
 };
 
 var existingInterval = null;
+// This is used to ensure the program throws on duplicate metrics during first run
+// We might want to consider not supporting running the default metrics function more than once
```

Comment:
That's a (bigger) breaking change, though. But makes it more in line with the official clients